week2_shoppinglist/index.js

```diff
@@ -0,0 +1,384 @@
+        src="./images/logo.png"
+        alt="쇼핑몰 로고"
+        width="36rem"
+        onclick="goHome()"
+      />
+      <h1 lang="en">SHOPPING LIST</h1>
+      <img src="./images/menu.svg" alt="햄버거" class="menuBtn" onclick="openNav()" />
+    </header>
+    <nav class="sidebar">
+      <div class="menuBtn" onclick="closeNav()">x
+      </div>
+      <ul>
+        <li>관심 상품 목록</li>
+        <li>관리자 페이지</li>
+        <li><a href="cart.html">장바구니</a></li>
+      </ul>
+    </nav>
+      `;
+  }
+}
+customElements.define("shop-header", HeaderComponent); // html 태그 정의 및 구현
+
+// home 이동
+function goHome() {
+  location.href = "index.html";
+}
+const sidebar = document.querySelector(".sidebar");
+// 사이드바 오픈
+function openNav() {
+  sidebar.style.transform = "translateX(0)";
+}
+// 사이드바 닫기
+function closeNav() {
+  sidebar.style.transform = "translateX(100%)";
+}
+
+/* index */
+
+// 상품 리스트
+const SHOPPING_LIST = [
+  {
+    id: 1,
+    category: "stationery",
+    image: "./images/note.jpg",
+    name: "잔망루피 페이스 스프링 노트",
+    price: 4000,
+    liked: false,
+  },
+  {
+    id: 2,
+    category: "stationery",
+    image: "./images/bag.jpg",
+    name: "잔망루피 쇼핑백 (2종/택1)",
+    price: 1500,
+    liked: false,
+  },
+  {
+    id: 3,
+    category: "stationery",
+    image: "./images/sticker.jpg",
+    name: "잔망루피 스티커",
+    price: 1500,
+    liked: false,
+  },
+  {
+    id: 4,
+    category: "fashion",
+    image: "./images/doll.jpg",
+    name: "잔망루피 파자마 봉제 인형",
+    price: 29000,
+    liked: false,
+  },
+  {
+    id: 5,
+    category: "fashion",
+    image: "./images/passport.jpg",
+    name: "잔망루피 여권 케이스",
+    price: 5900,
+    liked: false,
+  },
+  {
+    id: 6,
+    category: "fashion",
+    image: "./images/flower.jpg",
+    name: "잔망루피 꽃다발 봉제 인형",
+    price: 34000,
+    liked: false,
+  },
+  {
+    id: 7,
+    category: "digital",
+    image: "./images/mouse.jpg",
+    name: "잔망루피 마우스패드",
+    price: 4000,
+    liked: false,
+  },
+  {
+    id: 8,
+    category: "digital",
+    image: "./images/usb.jpg",
+    name: "로이체 잔망루피 5in1 멀티 USB 허브",
+    price: 36900,
+    liked: false,
+  },
+  {
+    id: 9,
+    category: "digital",
+    image: "./images/mic.jpg",
+    name: "로이체 잔망루피 블루투스 마이크 스피커",
+    price: 39900,
+    liked: false,
+  },
+];
+
+// 상품 리스트 렌더링
+function renderItems(filterCategory = "all") {
+  const filteredItems = SHOPPING_LIST.filter((item) => {
+    return filterCategory === "all" || item.category === filterCategory; // 상품 카테고리 필터링
+  });
+
+  const itemsContainer = document.querySelector(".items"); // 상품 리스트를 렌더링할 class 가져오기
+  itemsContainer.innerHTML = ""; // 기존 상품 목록 비우기
+
+  // 상품 목록 추가
+  filteredItems.forEach((item) => {
+    const article = document.createElement("article"); // 각 상품에 대한 article 생성
+    article.className = "item";
+    article.onclick = function () {
+      addCart(item);
+    };
+
+    // 이미지
+    const img = document.createElement("img");
+    img.src = item.image;
+    img.alt = "상품 이미지";
```

Comment:
여기 상품 리스트 전체에 alt 속성이 "상품 이미지"라는 string으로 들어가고있어!!! 
이 부분은 검색엔진이나 스크린 리더에서도 중요하게 사용되는 부분이여서 아래처럼 바꾸는 건 어떨까? ㅎㅎ
```suggestion
    img.alt = item.name;
```
참고: https://rgy0409.tistory.com/2884

Reply:
아...!!!! 이런 방법이 있었네요!! 감사합니다🙇🏻‍♀️🙇🏻‍♀️